cfme/automate/import_export.py

```diff
@@ -1,70 +1,112 @@
 # -*- coding: utf-8 -*-
+import attr
+from navmazing import NavigateToAttribute
+from widgetastic.widget import Checkbox
+from widgetastic.widget import FileInput
+from widgetastic.widget import Text
+from widgetastic.widget import View
 from widgetastic_patternfly import BootstrapSelect
 from widgetastic_patternfly import Button
+from widgetastic_patternfly import FlashMessages
+from widgetastic_patternfly import Input
 
-from cfme.base.ui import AutomateImportExportBaseView
-from cfme.base.ui import AutomateImportExportView
-from cfme.utils.appliance import Navigatable
+from cfme.base.login import BaseLoggedInPage
+from cfme.modeling.base import BaseCollection
+from cfme.modeling.base import BaseEntity
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep
 from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.appliance.implementations.ui import navigator
 
 
-class GitImportSelectorView(AutomateImportExportBaseView):
-    type = BootstrapSelect('branch_or_tag')
-    branch = BootstrapSelect(locator='.//div[contains(@class, "bootstrap-select git-branches")]')
-    tag = BootstrapSelect(locator='.//div[contains(@class, "bootstrap-select git-tags")]')
+class AutomateImportExportBaseView(BaseLoggedInPage):
+    flash = FlashMessages("div.import-flash-message")
+    title = Text('.//div[@id="main-content"]//h1')
 
-    submit = Button('Submit')
-    cancel = Button('Cancel')
+    @property
+    def in_import_export(self):
```

Comment:
We can name this directly as is_displayed

Reply:
Actually, this method indicate that are you in `import_export` page or not? basically, every view of import export will call this internally. `is_displayed` is for specific views.